source/Weapon.cpp

```diff
@@ -113,6 +113,8 @@ void Weapon::LoadWeapon(const DataNode &node)
 					child.PrintTrace("Skipping unknown or incomplete submunition attribute:");
 			}
 		}
+		else if(key == "fade out" && child.Size() >= 2)
+			fadeOut = child.Value(1);
```

Comment:
Move this down next to the other value reads (next to the lifetime ones looks good to me), and clamp it so that it's never negative.

Reply:
done